entrypoint.rb

```diff
@@ -10,12 +10,13 @@
 verbose = ENV["VERBOSE"]
 wait = ENV["WAIT_INTERVAL"]
 workflow_name = ENV["RUNNING_WORKFLOW_NAME"]
+api_endpoint = ENV['API_ENDPOINT'] || 'https://api.github.com'
```

Comment:
⚠️ If `ENV['API_ENDPOINT']` is `""` then it's not falsy, so the `||` won't work as expected (like when it's `nil`).

That's a problem because `Octokit::Client.new(api_endpoint: "")` won't work.

```suggestion
api_endpoint = ENV.fetch('API_ENDPOINT', "").strip.empty? ? "https://api.github.com" : ENV['API_ENDPOINT']
```

Reply:
Ahh kk sorry about that. I tested this out and it worked on GHE as well